parser/sectionParser.go

```
@@ -90,16 +110,23 @@ func getAcademicSession(rowInfo map[string]*goquery.Selection) schema.AcademicSe
 	return session
 }
 
+// getSectionNumber returns the matched value from a sectionPrefixRegexp on
+// `classInfo["Class Section:"]`. It expects classInfo to contain "Class Section:" key.
+// If there are no matches, an empty string returned.
 func getSectionNumber(classInfo map[string]string) string {
 	if syllabus, ok := classInfo["Class Section:"]; ok {
 		matches := sectionPrefixRegexp.FindStringSubmatch(syllabus)
+		// make sure it matches the correct format but only kep the section number
 		if len(matches) == 2 {
 			return matches[1]
 		}
 	}
 	return ""
```

Sections should always have a section number -- we should panic here instead of silently failing with an empty string